.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,60 @@
+name: Bug report
+description: You've found a bug on the documentation website
+labels: ['bug']
+body:
+  - type: dropdown
+    attributes:
+      label: What browser are you using?
```

Comment:
I've converted the browser question to a dropdown menu. Do we want to list all browsers, or just the most popular ones like:

- Firefox
- Chrome
- Edge
- Safari on macOS
- Safari on IOS
- Android browser
- Other <-- capture all other browsers